### PR TITLE
Adding Tests for Coveralls

### PR DESCRIPTION
**Pull request by seanlowjk**
Adding Simple Tests to increase Test Coverage

### CODE REVIEW

File: src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java [hunk 1]
```diff
@@ -167,4 +170,53 @@ public void parseTags_collectionWithValidTags_returnsTagSet() throws Exception {
...
+        assertThrows(ParseException.class, () -> ParserUtil.parseCategories(Arrays.asList(INVALID_CATEGORY)));
+    }
+
+    @Test
+    public void parse_collectionWithValidTags_returnsTagSet() {
+        String validOpeningHours = "0000 0001";
+        String invalidOpeningHours = "abcd efgh";
+        try {
+            assertTrue(ParserUtil.parseOpeningHours(validOpeningHours).equals(new OpeningHours(validOpeningHours)));
+            assertTrue(ParserUtil.parseOpeningHours(invalidOpeningHours).equals(new OpeningHours(invalidOpeningHours)));
```
Comment (jon-chua) [inline src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java]: i think you can use `assertParseFailure` for this!

File: src/test/java/seedu/savenus/logic/commands/BuyCommandTest.java [hunk 2]
```diff
@@ -17,6 +29,9 @@
     // history separately...
     // Please refer to the wallet and purchase history tests
 
+    private Model model = new ModelManager(getTypicalMenu(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalMenu(), new UserPrefs());
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply src/test/java/seedu/savenus/logic/commands/BuyCommandTest.java]: Will be fixed in the next commit!

File: src/test/java/seedu/savenus/logic/commands/BuyCommandTest.java [hunk 3]
```diff
@@ -38,4 +53,25 @@ public void equals() {
...
+        BuyCommand infiniteBuyCommand = new BuyCommand(Index.fromZeroBased(Integer.MAX_VALUE));
+        assertThrows(CommandException.class, () -> infiniteBuyCommand.execute(model));
+    }
+
+    @Test
+    public void execute_success() {
+        Purchase purchaseToAdd = new Purchase(CARBONARA.getName(), CARBONARA.getPrice());
+        String expectedMessage = String.format(BuyCommand.MESSAGE_BUY_FOOD_SUCCESS, purchaseToAdd.toString());
+        BuyCommand correctCommand = new BuyCommand(INDEX_FIRST_FOOD);
+        try {
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply src/test/java/seedu/savenus/logic/commands/BuyCommandTest.java]: Will be fixed in the next commit!

File: src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java [hunk 4]
```diff
@@ -167,4 +170,53 @@ public void parseTags_collectionWithValidTags_returnsTagSet() throws Exception {
...
+    @Test
+    public void parseCategories_collectionWithInvalidLocations_throwsParseException() {
+        assertThrows(ParseException.class, () -> ParserUtil.parseCategories(Arrays.asList(INVALID_CATEGORY)));
+    }
+
+    @Test
+    public void parse_collectionWithValidTags_returnsTagSet() {
+        String validOpeningHours = "0000 0001";
+        String invalidOpeningHours = "abcd efgh";
+        try {
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java]: Will be fixed in the next commit!

File: src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java (hunk 1 above)
Comment (seanlowjk) [reply src/test/java/seedu/savenus/logic/parser/ParserUtilTest.java]: Will be fixed in the next commit!

**Pull request merged**